packages/kotti-ui/source/kotti-button/types.ts

```diff
@@ -24,6 +24,13 @@ export namespace KottiButton {
 	}
 	export const iconPosition = z.nativeEnum(IconPosition)
 
+	export enum ToggleStatus {
+		ON = 'on',
+		OFF = 'off',
```

Comment:
```suggestion
		ON = 'ON',
		OFF = 'OFF',
```

Reply:
the other enums defined on this file are in lowercase, I'm fine having them either lowercased or uppercased but I wouldn't mix.